extensions/twitter/src/components/tweet.tsx

```diff
@@ -1,8 +1,8 @@
 import { ActionPanel, Detail, Image, ImageMask, List, OpenInBrowserAction, showToast, ToastStyle } from "@raycast/api";
-import millify from "millify";
+//import millify from "millify";
```

Comment:
NIT: Better to remove instead of commenting out.

Reply:
🤦